cmd/cql/internal/drop.go

```diff
@@ -0,0 +1,57 @@
+package internal
+
+import (
+	"github.com/CovenantSQL/CovenantSQL/client"
+)
+
+var CmdDrop = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdDrop should have comment or be unexported